3-enrich/emr-etl-runner/Gemfile.lock

```diff
@@ -1,67 +1,138 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    CFPropertyList (2.3.2)
+    CFPropertyList (2.3.5)
+    addressable (2.5.0)
+      public_suffix (~> 2.0, >= 2.0.2)
+    avro (1.8.1)
+      multi_json
     awrence (0.1.0)
-    builder (3.2.2)
+    builder (3.2.3)
     contracts (0.11.0)
-    coveralls (0.8.15)
+    coveralls (0.8.19)
       json (>= 1.8, < 3)
       simplecov (~> 0.12.0)
       term-ansicolor (~> 1.3)
       thor (~> 0.19.1)
-      tins (>= 1.6.0, < 2)
-    diff-lcs (1.2.5)
+      tins (~> 1.6)
+    diff-lcs (1.3)
     docile (1.1.5)
-    domain_name (0.5.20160826)
+    domain_name (0.5.20170223)
       unf (>= 0.0.5, < 1.0.0)
-    elasticity (6.0.10)
+    elasticity (6.0.12)
       fog (~> 1.0)
       rest-client (~> 1.0)
       unf (~> 0.1)
-    excon (0.52.0)
+    excon (0.55.0)
     fission (0.5.0)
       CFPropertyList (~> 2.2)
-    fog (1.25.0)
+    fog (1.38.0)
+      fog-aliyun (>= 0.1.0)
```

Comment:
Building on your remark, I think I'll open a PR against elasticity specifying that we only depend on `fog-aws` that'll avoid all those unnecessary deps.